pubspec.yaml

```diff
@@ -49,7 +49,13 @@ dependencies:
 dev_dependencies:
   flutter_test:
     sdk: flutter
-  test: any  
+  test: any
+  flutter_launcher_icons: ^0.7.4
+
+flutter_icons:
+  android: false
```

Comment:
Why android false and iOS true? Would icon still be used on both android and iOS?

Reply:
I made android false so that I can add Android icon manually. I wanted to use the adaptive icon which cannot be done by `flutter_launcher_icons` dependency.